Makefile

```
@@ -321,8 +321,8 @@ helm: ## Download helm locally if necessary.
 GENREF = $(PROJECT_DIR)/bin/genref
 .PHONY: genref
 genref: ## Download genref locally if necessary.
-	# TODO: update the package version. "c519be1" - is the master tip on 27-Oct-2023
-	@GOBIN=$(PROJECT_DIR)/bin $(GO_CMD) install github.com/kubernetes-sigs/reference-docs/genref@c519be1
+	# TODO: update the package version. "c519be1" - is the master tip on 30-Oct-2023
+	@GOBIN=$(PROJECT_DIR)/bin $(GO_CMD) install github.com/kubernetes-sigs/reference-docs/genref@23a78f4
```

```suggestion
	@GOBIN=$(PROJECT_DIR)/bin $(GO_CMD) install github.com/kubernetes-sigs/reference-docs/genref@v0.28.0
```

Can we use the released version?

https://github.com/kubernetes-sigs/reference-docs/releases/tag/v0.28.0

This version seems to have your patch.